python/tvm/relay/op/transform.py

```diff
@@ -308,7 +308,7 @@ def scatter_add(data, indices, updates, axis):
     return _make.scatter_add(data, indices, updates, axis)
 
 
-def reshape_like(data, shape_like):
+def reshape_like(data, shape_like, lhs_begin=0, lhs_end=None, rhs_begin=0, rhs_end=None):
```

Comment:
Could you add the additional parameters (lhs_begin, lhs_end, rhs_begin, rhs_end) in the docstring description?